plone/app/content/browser/contents/__init__.py

```diff
@@ -146,26 +146,43 @@ def get_actions(self):
     def get_columns(self):
         # Base set of columns
         columns = {
-            'id': translate(_('ID'), context=self.request),
-            'ModificationDate': translate(_('Last modified'), context=self.request),  # noqa
-            'EffectiveDate': translate(_('Publication date'), context=self.request),  # noqa
-            'ExpirationDate': translate(_('Expiration date'), context=self.request),  #noqa
             'CreationDate': translate(_('Created on'), context=self.request),  # noqa
-            'review_state': translate(_('Review state'), context=self.request),  # noqa
-            'Subject': translate(_('Tags'), context=self.request),
-            'portal_type': translate(_('Type'), context=self.request),
-            'is_folderish': translate(_('Folder'), context=self.request),
+            'Creator': translate(_('Creator'), context=self.request),
```

Comment:
Ah. Below, at https://github.com/plone/plone.app.content/pull/78/files#diff-5dba3bbf58278abc3e87451c188e85d3R31 the ``plone.app.vocabularies.Catalog`` is only secured by the ``View`` permission. Normal users shouldn't be able to see the creators of a content item easily.  So this *is* a security problem.
I will solve that by securing each of the ``_unsafe_metadata`` attributes by a harder permission - probably ``Modify portal content``.